packages/core/admin/admin/src/pages/Users/ListPage/PaginationFooter/PageSize.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Row, Select, Option, Text } from '@strapi/parts';
+import { useQueryParams } from '@strapi/helper-plugin';
+import styled from 'styled-components';
+
+const StyledText = styled(Text)`
+  margin-left: 5px;
+`;
+
+const PageSize = () => {
+  const [
+    {
+      query: { pageSize },
+    },
+    setQuery,
+  ] = useQueryParams();
+  const handleChange = e => {
+    setQuery({
+      pageSize: e,
+      page: 1,
+    });
+  };
+
+  return (
+    <Row>
+      <Select onChange={handleChange} value={pageSize}>
+        <Option value="10">10</Option>
+        <Option value="20">20</Option>
+        <Option value="50">50</Option>
+        <Option value="100">100</Option>
+      </Select>
+      <StyledText textColor="neutral500" as="label">
```

Comment:
Can you use the Text component please wrapped with a Box to avoid custom css. A label without a target element seems to be useless. You will need to add an id to the select and a htmlFor props to the label.  BTW, the color contrast between the text and the white background is too low. Can you use the neutral600 color please.


```jsx
<Row>
  <Select id="page-size" onChange={handleChange} value={pageSize}>
    <Option value="10">10</Option>
    <Option value="20">20</Option>
    <Option value="50">50</Option>
    <Option value="100">100</Option>
  </Select>
  <Box paddingLeft={2}>
    <Text textColor="neutral600" as="label" htmlFor="page-size">Entries per page</Text>
  </Box>
</Row>
```